dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/messages.nlsprops

```diff
@@ -1273,11 +1270,22 @@ jsp.isthreadsafe.warning=CWWJS0003W: The Pages 3.1 specification strongly advise
 jsp.isthreadsafe.warning.explanation=SingleThreadModel was removed from Servlet 6.0. Pages 3.1 deprecated isThreadSafe with plans to remove it in a future Pages update.
 jsp.isthreadsafe.warning.useraction=Remove the isThreadSafe directive from all pages.
 
-
 jsp.useinmemory.warning=CWWJS0004W: The 'useInMemory' JSP Option is not supported. The JSPs will be compiled and saved on disk instead.
 jsp.useinmemory.warning.explanation=This feature was never ported from Traditional WebSphere for historical reasons.
 jsp.useinmemory.warning.useraction=Remove the useInMemory option from the server.xml, ibm-web-bnd.xml, or the ibm-web-ext.xmi files.
 
 pages.removed.element.error=CWWJS0005E: The {0} element was removed in Pages 4.0 and therefore is no longer supported.
 pages.removed.element.error.explanation=The jsp plugin and related actions were deprecated in Pages 3.1 and removed in 4.0. 
 pages.removed.element.error.useraction= Remove the jsp plugin, fallback, and params elements from your web app.
+
+jsp.javasourcelevel.warning=CWWJS0006W: The specified javaSourceLevel={0} value exceeds the {1} installed Java level. Therefore, the default Java level is set to {1}.
```

Comment:
The warning and explanation would be better if they started with "The"
- "The {0} value..."
- "The file cannot ..."